tests/qa_automation/execute_test_run.pm

```diff
@@ -21,8 +21,13 @@ use upload_system_log;
 use base "opensusebasetest";
 
 sub run {
-    my $timeout = get_var("MAX_JOB_TIME") || 7200;
-    my $test    = get_var("QA_TESTSUITE") . get_var("QA_VERSION");
+    my $timeout   = get_var("MAX_JOB_TIME") || 7200;
+    my $test      = get_var("QA_TESTSUITE") . get_var("QA_VERSION");
+    my $runfile   = "/usr/share/qa/tools/test_$test-run";
+    my $run_exist = script_run("ls $runfile | wc -l");
```

Comment:
Why not to use bash test command here (e.g. '[ ! -f /tmp/foo.txt ]')? Then next condition can be simplified as we care only if return code is zero or anything else.

Reply:
I tried to use some bash like that, but it turn to check the file in host machine, not in running VM. So I change to this way.

Reply:
hmm, that's weird, because if you wrap it in script_run it should always be executed in SUT. I'll merge it as per @SeroSun comment, but please make a note somewhere to try more scalable solution, maybe =)

Reply:
@rwx788 copy that, will included in next update.